simapp/v2/simdv2/cmd/commands.go

```diff
@@ -176,13 +176,13 @@ func appExport[T transaction.Tx](
 
 	var simApp *simapp.SimApp[T]
 	if height != -1 {
-		simApp = simapp.NewSimApp[T](logger, viper)
+		simApp = simapp.NewSimApp[T](logger, viper, root.NewBuilder())
```

Comment:
i don't think that will work great, it store home will be the current directory, while the app will have `simapp.DefaultNodeHome`. It will miss the store config as well

Reply:
nvm, it's late lol

Reply:
nvm, it is built in NewSimapp